cmd/nerdctl/run_security_linux_test.go

```diff
@@ -190,3 +191,37 @@ func TestRunSeccompCapSysPtrace(t *testing.T) {
 	base.Cmd("run", "--rm", "--cap-add", "sys_ptrace", testutil.AlpineImage, "sh", "-euxc", "apk add -q strace && strace true").AssertOK()
 	// Docker/Moby 's seccomp profile allows ptrace(2) by default, but containerd does not (yet): https://github.com/containerd/containerd/issues/6802
 }
+
+func TestRunPrivileged(t *testing.T) {
+	// docker does not support --privileged-without-host-devices
+	testutil.DockerIncompatible(t)
+
+	base := testutil.NewBase(t)
+
+	devPath := "/dev/dummy-zero"
```

Comment:
Q (for my own understanding): Why not re-use /dev/zero or /dev/urandom? Why do we need to create a new device?

Reply:
Choosing a nonstandard device to ensure it will not be affected by different OSes/Environments that this test running on.

And also as https://github.com/opencontainers/runc/blob/d11f4d756e85ece5cdba8bb69f8bd4db3cdcbeab/libcontainer/specconv/spec_linux.go#L173-L174 has shown, `/dev/zero` and/or `/dev/random` are standard devices that will be created for containers whether this `without-host-device` is specified or not.